optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticType.java

```diff
@@ -23,4 +23,9 @@
      */
     public String name();
 
+    /**
+     * @return never null
+     */
+    public String formattedName();
```

Comment:
Good point on B). For benchmarks names we have no choice, as Japanese users even use japanese characters (and that works) to name them.

Let's call that proposal C) to keep name() and rename getFormattedName() to getLabel(). That class was originally an enum, which explains.

+1 on all 3 proposals A), B) and C) - which is probably very helpful :) - your call which to go for.